improver_tests/lapse_rate/test_LapseRate.py

```diff
@@ -108,6 +108,25 @@ def test_returns_expected_values(self):
             self.orography)
         self.assertArrayAlmostEqual(result, expected_out)
 
+    def test_returns_expected_values_generator(self):
+        """Test that the function returns True at points where the height
+           difference to the central pixel is greater than 35m."""
+
+        expected_out = np.array(
+            [[True, True, False, False, False, False, False, False, True],
+             [True, True, False, False, False, False, False, False, True]])
+
+        result = np.zeros(expected_out.shape, dtype=bool)
```

Comment:
Suggest using `np.zeros_like` to more clearly express the intention of creating an array the same shape as an existing array.

Reply:
Okay